### PR TITLE
CHE-2286: disable menu items until the workspace is not running

### PR DESCRIPTION
**Pull request by olexii4**
### What does this PR do?

Disable menu items until the workspace is not running
### What issues does this PR fix or reference?
#2286
### Major change checklist
- [x] Tests passed

@ashumilova, @vparfonov please review

Signed-off-by: Oleksii Orel oorel@codenvy.com

### CODE REVIEW

File: ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/app/AppContext.java [hunk 1]
```diff
@@ -187,13 +188,24 @@
     
     String getWorkspaceId();
 
+    /**
+     * Returns {@link Workspace} instance of current workspace.
+     *
+     * @return current workspace
+     */
+/*    Workspace getWorkspace();*/
```
Comment (benoitf) [inline ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/app/AppContext.java]: do we need this commented method ?

Comment (olexii4) [reply ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/app/AppContext.java]: fixed

**Review by ashumilova — COMMENTED** [review]
As far as all menu items should be disabled - please make sure this fix will also work for plugins :
Openshift menu https://github.com/codenvy/plugin-openshift
Artik menu https://github.com/codenvy/artik-ide

**Comment by olexii4** [comment]
https://github.com/eclipse/che/pull/2473
https://github.com/codenvy/artik-ide/pull/156

**Review by benoitf — APPROVED** [review]

**Comment by olexii4** [comment]
https://github.com/codenvy/plugin-openshift/pull/37

**Review by ashumilova — APPROVED** [review]

**Pull request merged**